src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -3354,7 +3345,7 @@ void WriteTableOfContents(EnergyPlusData &state)
                         for (int jTable = 1; jTable <= ort->OutputTableBinned(iInput).numTables; ++jTable) {
                             int const curTable = ort->OutputTableBinned(iInput).resIndex + (jTable - 1);
                             std::string curName;
-                            if (ort->unitsStyle == UnitsStyle::InchPound) {
+                            if ((ort->unitsStyle == UnitsStyle::InchPound) || (ort->unitsStyle == UnitsStyle::InchPoundExceptElectricity)) {
```

Comment:
This repeated pattern is probably fine.  I'm inclined to say you could add a little worker method to the ort class to clean this code up.  Consider this example:

```
#include <iostream>

enum class UnitsStyle {InchPound, InchPoundExceptElectricity, SI};

struct ORT {
    UnitsStyle unitsStyle;
    ORT(UnitsStyle us) : unitsStyle(us) {}
    bool ip() const {
        return this->unitsStyle == UnitsStyle::InchPound || this->unitsStyle == UnitsStyle::InchPoundExceptElectricity;
    }
};

int main() {
    ORT ort1(UnitsStyle::InchPound);
    std::cout << ort1.ip() << std::endl;
    ORT ort2(UnitsStyle::InchPoundExceptElectricity);
    std::cout << ort2.ip() << std::endl;
    ORT ort3(UnitsStyle::SI);
    std::cout << ort3.ip() << std::endl;
}
```

That way all of these if checks would just be replaced with:

```
if (ort->ip()) {
```

Which is a lot less to digest.  Notes:

- I didn't look up the actual ort struct name, that's just a quick example
- I'm not saying `ip` is really the right function name, just a quick guess 

Do you think this change is worth it here?

Reply:
I think I got the idea here. Probably worth a try---except the "ORT" struct name could be a little bit confusing here as there are many similar names of "ort" or something alike that are already used in this(these) file(s).

Reply:
The ORT is just a dummy name I put in that sample code.  Take the `ort` variable...it is some type.  Is it an instance of a `struct OutputReportTabular`?  If so, then my example is meant to add a method as such:

```
struct OutputReportTabular {
  bool ip() const {
    /// etc.
  }
}
```

Basically - add the new method to whatever struct the `ort` variable is.